languages/ocaml.scm

```diff
@@ -1100,6 +1104,20 @@
   )
 )
 
+(attribute_payload
+  (function_type
+    "->" @append_indent_start
+    (_) @append_indent_end
+    .
+  )
+)
```

Comment:
Isn't this query causing the double indentation? I think it can be removed

Reply:
Done in 40e20efa6a1